README.md

```diff
@@ -116,6 +116,8 @@ carthage update Box Result
 
 If the framework you want to add to your project has dependencies explicitly listed in a [Cartfile][], Carthage will automatically retrieve them for you. You will then have to **drag them yourself into your project** from the [Carthage/Build] folder.
 
+If the embedded framework in your project has dependencies to other frameworks you must list them in the [Cartfile][], and **link them to application target** (even if application target does not have dependency to that frameworks and never uses them).
```

Comment:
But initially it has to be put to `Cartfile`, hasn't it? Otherwise, I don't understand, how it would be added to `Cartfile.resolved`.

For example, in this issue #1401 I described my **Setup**. In order to pull and build **Other.framework** I have to put it into `Cartfile` initially. After that, the record is added to `Cartfile.resolved`. 

Other developers then, could `bootstrap` it and get all dependancies ready to be used. 

What I am missing?

Note: the project I am talking about is here: https://github.com/evgeniyd/Giraffe